File: Easy/python/palindrome_check.py
def is_palindrome(string):
    formatted = ''.join([char.lower() for char in string if char.isalnum()])
    return formatted == formatted[::-1]


print(is_palindrome("race car"))  # true
print(is_palindrome("A man, a plan, a canal. Panama"))  # true
print(is_palindrome("never odd or even"))  # true
print(is_palindrome("0_0 (: /- :) 0–0"))  # true
print(is_palindrome("My age is 0, 0 si ega ym."))  # true
print(is_palindrome("nope"))  # false
print(is_palindrome("not a palindrome"))  # false
print(is_palindrome("almostomla"))  # false
print(is_palindrome("1 eye for of 1 eye."))  # false

File: Easy/js/palindromeCheck.js
const palindromeCheck = str => {
  const reg = /[\W_]/g;
  const formatted = str.replace(reg, "").toLowerCase();
  return (
    formatted ===
    formatted
      .split("")
      .reverse()
      .join("")
  );
};

console.log(palindromeCheck("race car")); // true
console.log(palindromeCheck("A man, a plan, a canal. Panama")); // true
console.log(palindromeCheck("never odd or even")); // true
console.log(palindromeCheck("0_0 (: /- :) 0–0")); // true
console.log(palindromeCheck("My age is 0, 0 si ega ym.")); // true
console.log(palindromeCheck("nope")); // false
console.log(palindromeCheck("not a palindrome")); // false
console.log(palindromeCheck("almostomla")); // false
console.log(palindromeCheck("1 eye for of 1 eye.")); // false
